### ISSUE TITLE
Poor training performance of delg

### ISSUE THREAD

**Issue by hewumars** [tensorflow/models#9362]
parameters：
--train_file_pattern=/home/data_ssd/hewu/vehicle_reid/train*
--validation_file_pattern=/home/data_ssd/hewu/vehicle_reid/validation*
--dataset_version=gld_v2_clean
--logdir=hz_training_2
--batch_size=256
--max_iters=150000
--initial_lr=0.01

hardware: 2080Ti GPU x8

commit-id: 85f5292

![image](https://user-images.githubusercontent.com/7970022/95697791-74c7fa80-0c72-11eb-9ae7-f275faa1aa0d.png)

![image](https://user-images.githubusercontent.com/7970022/95697803-7ee9f900-0c72-11eb-98b6-ed98b71b359d.png)

training accuracy  is always close to 0.
